### PR TITLE
Add coursier based installation instructions

### LINKED ISSUE #545: Conscript fails to install giter8 on MacOS
## steps

Trying to install giter8 locally using conscript (since `brew` no longer will, see also https://github.com/foundweekends/giter8/issues/475).

MacOS 11.2.3, conscript 0.5.5, sbt 1.5.0.

## problem

giter8 won't download, let alone install.

<details>

<summary>stack trace</summary>

```zsh
% cs foundweekends/giter8
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.jboss.netty.util.internal.ByteBufferUtil (file:/Users/jasperstein/.conscript/boot/scala-2.11.12/org.foundweekends.conscript/conscript/0.5.5/netty-3.9.2.Final.jar) to method java.nio.DirectByteBuffer.cleaner()
WARNING: Please consider reporting this to the maintainers of org.jboss.netty.util.internal.ByteBufferUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
An unexpected error occurred: Please check your internet connection.
Exception message: https://api.github.com/repos/foundweekends/giter8/git/refs/heads/master
```

</details>

## expectation

giter8 is installed

### PR DESCRIPTION
**Pull request by TonioGela**
I added the Coursier based installation instructions, removing the Homebrew ones since the formula was disabled (see #475). I focused on the name clash between Conscript and Coursier, but there wasn't an obvious solution for that, so I just pointed out the possible issue. 

Please review my English. 😓 
Closes #545.

### CODE REVIEW

**Comment by TonioGela** [comment]
@eed3si9n or @xuwei-k

File: docs/01.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ out: setup.html
 Setup
 -----
 
+#### Conscript
```
Comment (eed3si9n) [inline docs/01.md]: I think we can remove the Conscript section.

Instead we can add curling from Maven Central?
https://github.com/foundweekends/giter8/issues/454

Comment (TonioGela) [reply docs/01.md]: Will do!

Comment (TonioGela) [reply docs/01.md]: Added curl here: https://github.com/foundweekends/giter8/pull/565/files#diff-7b69ef160770805d36336cee7afbf5e566ef54b684bf5c2bc7ed48553e653f1bR24

**Comment by TonioGela** [comment]
Closes #454

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Comment by TonioGela** [comment]
Lol, this should close 545 and 454

**Pull request merged**